querybook/webapp/redux/dataDoc/reducer.ts

```diff
@@ -20,8 +20,32 @@ const initialState: Readonly<IDataDocState> = {
     sessionByDocId: {},
     editorsByDocIdUserId: {},
     accessRequestsByDocIdUserId: {},
+    dataDocWithSchema: {
+        docs: [],
+        total: 0,
+        page: 0,
+        pageSize: 10,
+        filtered: [{ value: '' }],
+    },
 };
 
+function getDataDocWithSchema(state = initialState.dataDocWithSchema, action) {
+    return produce(state, (draft) => {
+        switch (action.type) {
+            case '@@dataDoc/RECEIVE_DATA_WITH_SCHEMA': {
+                draft.docs = action.payload.docs;
+                draft.total = Math.ceil(
```

Comment:
why is this ceil done twice (i see the same math in action)? also total is totalPages instead of total?